### ISSUE TITLE
something wrong with 'make' command

### ISSUE THREAD

**Issue by showaichuan** [smallcorgi/Faster-RCNN_TF#126]
Hi ,when I build the Cython modules,it went wrong just like this ,

python setup.py build_ext --inplace
Traceback (most recent call last):
  File "setup.py", line 13, in <module>
    from Cython.Distutils import build_ext
ImportError: No module named Cython.Distutils
Makefile:2: recipe for target 'all' failed
make: *** [all] Error 1
I have installed cpython,but still went wrong,I would greatly appreciate it if someone could explain how to fix this issue.

**Comment by LHagendoorn** [comment]
Are you using anaconda? are you in the right enviroment?

**Comment by showaichuan** [comment]
Thanks for your advice.but after installing anaconda,I got another problem,
ImportError: No module named 'tensorflow'
nvcc fatal   : A single input file is required for a non-link phase when an outputfile is specified
g++: error: roi_pooling_op.cu.o: no such file or directory
g++: error: GOOGLE_CUDA=1: no such file or directory

I have installed tensorflow before,but when I import it from command line,it shows :
ImportError: No module named 'tensorflow'

but when I install tensorflow again,it shows : 
Requirement already satisfied: tensorflow-gpu in /usr/local/lib/python3.5/dist-packages
Requirement already satisfied: numpy>=1.11.0 in /usr/local/lib/python3.5/dist-packages (from tensorflow-gpu)
Requirement already satisfied: wheel>=0.26 in /usr/local/lib/python3.5/dist-packages (from tensorflow-gpu)
Requirement already satisfied: protobuf>=3.1.0 in /usr/local/lib/python3.5/dist-packages (from tensorflow-gpu)
Requirement already satisfied: six>=1.10.0 in /usr/lib/python3/dist-packages (from tensorflow-gpu)
Requirement already satisfied: setuptools in /usr/lib/python3/dist-packages (from protobuf>=3.1.0->tensorflow-gpu)

so,what should I do now to solve this issue?

**Comment by ghost** [comment]
pip install cython

**Comment by LHagendoorn** [comment]
Oh I meant it more as a question, though using anaconda is always a good idea!
Anaconda allows you to create various enviroments, each with their own packages and setup. As such, you have to make sure you are installing your packages in the enviroment, and that you are also running your code from there.

Other than that: do you have an nvidia gpu? are you running python 2.7? Are you using linux?

**Comment by showaichuan** [comment]
Yeah,you're right,I have an nvidia gpu with linux os,and run default python 2.7,I'm new to this field and have no idea what to do now,should I uninstall anaconda?but how to solve the previous problem?

By the way,I have installed easydict,but it still shows 'No module named easydict.'I'm really confused now.

**Comment by LHagendoorn** [comment]
https://www.tensorflow.org/install/install_linux#InstallingAnaconda

Perhaps this will help you, in your console, you should be in your (tensorflow) enviroment as shown there. Then you also have to make sure that you are running your script from within this enviroment. If you're in a conda env you can type: "conda list" Which allows you to check all the packages installed

**Comment by showaichuan** [comment]
@TheLaurensOK, I'll try,thank you so much for your advice!

**Comment by wingNine** [comment]
@showaichuan Have you solved your problem? I also have the same problem as you. How did you solve it?
